### PR TITLE
Milestone 1: linter configurations 

### PR DESCRIPTION
**Pull request by isume295**
In this milestone I:

1. added a linter.yml file

### CODE REVIEW

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @isume295    ,
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
## To Highlight :100:
- linters are passing ✔️ 

## Required Changes :recycle:

- [x] [Required] I would like to suggest that you add a readme file with this readme template from Microverse [Click Here >>](https://github.com/microverseinc/readme-template). Please make sure that the following sections are present and filled in correctly, only the Faq section is optional.

[About the Project](https://github.com/microverseinc/readme-template#about-project)
[Built With](https://github.com/microverseinc/readme-template#built-with)
[Tech Stack](https://github.com/microverseinc/readme-template#tech-stack)
[Key Features](https://github.com/microverseinc/readme-template#key-features)
[Live Demo](https://github.com/microverseinc/readme-template#live-demo)
[Getting Started](https://github.com/microverseinc/readme-template#getting-started)
[Setup](https://github.com/microverseinc/readme-template#setup)
[Prerequisites](https://github.com/microverseinc/readme-template#prerequisites)
[Install](https://github.com/microverseinc/readme-template#install)
[Usage](https://github.com/microverseinc/readme-template#usage)
[Run tests](https://github.com/microverseinc/readme-template#run-tests)
[Deployment](https://github.com/microverseinc/readme-template#triangular_flag_on_post-deployment)
[Authors](https://github.com/microverseinc/readme-template#authors)
[Future Features](https://github.com/microverseinc/readme-template#future-features)
[Contributing](https://github.com/microverseinc/readme-template#contributing)
[Show your support](https://github.com/microverseinc/readme-template#support)
[Acknowledgements](https://github.com/microverseinc/readme-template#acknowledgements)
[FAQ (OPTIONAL)](https://github.com/microverseinc/readme-template#faq)
[License](https://github.com/microverseinc/readme-template#license)

- [x] kindly consider adding meaning-full commit messages so as its easy to understand whats being done in a certain commit message remember they have to be in imperative form

_Check the comments under the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!:clap::clap::clap:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by isume295** [comment]
Thank you,

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi  @isume295 ☺️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

you-are-near

## Highlights ✔️
- [x] Good commit messages.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow

 ## Required Changes ♻️

- As stated by the previous reviewer, Kindly 😃 use [the Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. The only optional section is a FAQ. Please remove the template logo on the top, if you have one, you can replace it. This is crucial to have a professional and instructive repository. 👌

| Your project| 
| --- |
|    image  |

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -3,10 +3,10 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
+    <title>Microverse</title>
     <link rel="stylesheet" href="styles.css">
 </head>
 <body>
     <h1 class="headingStyle">Hello Microverse!</h1>
 </body>
```
Comment (Kamba56) [inline index.html:11]: - [Optional] COnisder using an indentation of two spaces in all your documents. This is a convention and also best practice.

**Review by Kamba56 — APPROVED** [review]
Hi @isume295, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (isume295) [reply index.html:11]: Okay I will consider, Thank you

**Pull request merged**